log.go

```diff
@@ -265,6 +271,9 @@ func callsite(flag uint32) (string, int) {
 // rules.
 func (b *Backend) print(lvl, tag string, args ...interface{}) {
 	t := time.Now() // get as early as possible
+	if b.flag&LUTC != 0 {
+		t = t.UTC()
+	}
```

Comment:
Will need to update `printf` below too I believe.

Reply:
good catch